src/common/SegmentedButton.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { ButtonStyle, getThemeColor, getReverseThemeColor } from "./Button";
+import ActionLink from './ActionLink';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCertificate } from "@fortawesome/free-solid-svg-icons";
+import Color from 'color';
+import classnames from "classnames";
+
+const SegmentedButton: React.FC<SegmentedButtonProps> = (props) => {
+  const [selected, setSelected] = useState(props.selected);
+  const THEME_COLOR = getThemeColor(props.theme);
+  const REVERSE_THEME_COLOR = getReverseThemeColor(props.theme);
+  const transparent = ButtonStyle.TRANSPARENT == props.theme;
+  return (
+    <div className={classnames("segmented-button", {})}>
+      {props.options.map((option) => (
+        <div className={classnames("segmented-button-option", selected == option.id ? "selected" : {})} key={"" + option.id}>
+          {option.updates && (
+            <div className="updates">
+              {option.updates === true ? (
+                <FontAwesomeIcon icon={faCertificate} />
+              ) : option.updates == Infinity ? (
+                "∞"
+              ) : (
+                option.updates
+              )}
+            </div>
+          )}
+          <ActionLink
+            link={{ 
+              onClick: () => {
+                setSelected(option.id);
+                if (option.onClick) {
+                  option.onClick();
+                }
+              },
+              href: option.href,
+            }}
+            className="segmented-button-link"
+            allowDefault={!!option.href && !option.onClick}
+          >
+            {option.label}
+          </ActionLink>
+        </div>
+      ))}
+      <style jsx>{`
+        .segmented-button {
+          border-radius: 25px;
+          background-color: ${REVERSE_THEME_COLOR};
+          border: 2px solid ${props.color || THEME_COLOR};
+          display: flex;
+          justify-content: space-evenly;
+        }
+        .segmented-button-option {
+          display: inline-block;
+          position: relative;
+        }
+        .segmented-button-option > :global(button),
```

Comment:
Small improvement: rather than using :global, you can use css.resolve like here: https://github.com/essential-randomness/bobaboard-ui/blob/master/src/common/PopupButtons.tsx#L30